yarn-project/simulator/src/avm/opcodes/external_calls.ts

```diff
@@ -61,15 +67,27 @@ abstract class ExternalCall extends Instruction {
     const totalGas = sumGas(this.gasCost(memoryOperations), allocatedGas);
     context.machineState.consumeGas(totalGas);
 
+    // TRANSITIONAL: This should be removed once the AVM is fully operational and the public executor is gone.
     const nestedContext = context.createNestedContractCallContext(
       callAddress.toFr(),
       calldata,
       allocatedGas,
       this.type,
       FunctionSelector.fromField(functionSelector),
     );
+    const pxContext = createPublicExecutionContext(nestedContext, calldata);
```

Comment:
Should the "TRANSITIONAL" comment be moved to before the pxContext initialization?

Reply:
Oh I left it there because the idea is that when migrating back, you can delete everything up to line 83 (the commented lines) and uncomment those.